### PR TITLE
docs: decapitalize IPFS where possible

### PR DESCRIPTION
**Pull request by RichardLitt**
Tried to check all instances of IPFS and make sure they werent referring to the CLI tool. See #2910.

License: MIT
Signed-off-by: Richard Littauer richard.littauer@gmail.com

### CODE REVIEW

File: core/commands/cat.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ var CatCmd = &cmds.Command{
 	},
 
 	Arguments: []cmds.Argument{
-		cmds.StringArg("ipfs-path", true, true, "The path to the IPFS object(s) to be outputted.").EnableStdin(),
+		cmds.StringArg("ipfs-path", true, true, "The path to the ipfs object(s) to be outputted.").EnableStdin(),
```
Comment (Kubuxu) [inline core/commands/cat.go]: In this case it should probably capital. It is about IPFS object.

File: core/commands/block.go [hunk 2]
```diff
@@ -103,15 +103,15 @@ It outputs to stdout, and <key> is a base58 encoded multihash.
...
+		Tagline: "Stores input as an ipfs block.",
 		ShortDescription: `
 'ipfs block put' is a plumbing command for storing raw ipfs blocks.
 It reads from stdin, and <key> is a base58 encoded multihash.
 `,
 	},
 
 	Arguments: []cmds.Argument{
-		cmds.FileArg("data", true, false, "The data to be stored as an IPFS block.").EnableStdin(),
+		cmds.FileArg("data", true, false, "The data to be stored as an ipfs block.").EnableStdin(),
```
Comment (Kubuxu) [inline core/commands/block.go]: Same as bellow.

**Comment by RichardLitt** [comment]
@Kubuxu updated

**Comment by Kubuxu** [comment]
SGTM but now you have merge conflict with master, you need to rebase.

**Comment by RichardLitt** [comment]
@Kubuxu rebased

File: core/commands/block.go [hunk 3]
```diff
@@ -43,10 +43,10 @@ multihash.
 
 var blockStatCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline: "Print information of a raw IPFS block.",
+		Tagline: "Print information of a raw ipfs block.",
```
Comment (whyrusleeping) [inline core/commands/block.go]: why is this one lowercase while the surrounding two are capitalized?

File: core/commands/root.go [hunk 4]
```diff
@@ -25,8 +25,8 @@ var Root = &cmds.Command{
 BASIC COMMANDS
   init          Initialize ipfs local configuration
   add <path>    Add a file to ipfs
-  cat <ref>     Show ipfs object data
-  get <ref>     Download ipfs objects
+  cat <ref>     Show IPFS object data
```
Comment (whyrusleeping) [inline core/commands/root.go]: why capitalized here where elsewhere youre making them lowercase in the taglines?

**Comment by whyrusleeping** [comment]
@RichardLitt youre gonna have to fix some of the tests. You changed print statements that we test the output of

**Comment by RichardLitt** [comment]
@whyrusleeping I tried to find the failing tests that are relevant to this, and I'm not seeing any. Can you help me?

**Comment by Kubuxu** [comment]
@RichardLitt t0020 - `ipfs init should not run while daemon is running` and `ipfs cat no repo message looks good`

When you open test console on circle, search for `not ok`.

**Comment by RichardLitt** [comment]
@Kubuxu I saw both of those: I don't know how to fix those particular issues. What code should I change to make that pass? How would I know? Can you please help me by listing exactly what you would do to fix one of these, here - I just don't know the workflow.

**Comment by whyrusleeping** [comment]
@RichardLitt take a look at the code for the test that is failing. They are in the test/sharness folder. The string next to the 'not ok' messages tells you which test. Look at the code being executed by the test and see why it would be failling.

**Comment by RichardLitt** [comment]
Ok:

1. I looked at `ipfs cat no repo message looks good`
2. I saw that it is making a command, and piping it to `cat_test_out`.
3. I replicate that by running:

```
test_expect_success "ipfs cat fails" '
    export IPFS_PATH="$(pwd)/.ipfs" &&
    test_must_fail ipfs cat Qmaa4Rw81a3a1VEx4LxB7HADUAXvZFhCoRdBzsMZyZmqHD 2> cat_fail_out
```

as

```
export IPFS_PATH="$(pwd)/.ipfs" &&
ipfs cat Qmaa4Rw81a3a1VEx4LxB7HADUAXvZFhCoRdBzsMZyZmqHD
```

It looks fine to me, and no different on master, where I run the same thing. What am I missing?

**Comment by RichardLitt** [comment]
@whyrusleeping Can you help me with my question above?

**Comment by RichardLitt** [comment]
@Kubuxu?

**Comment by whyrusleeping** [comment]
@RichardLitt The issue here is that you changed the capitalization of ipfs. in some tests
 in some tests we are expecting to see it lowercase, but now its capitalized. 

You will have to update these tests and change the capitalization the same way you did in the code.

While on this branch, cd into the test/sharness folder and run `make deps`. Then set `export TEST_NO_FUSE=1` to skip running the fuse tests (they likely won't run on your machine anyways), and `export TEST_VERBOSE=1` to see verbose test output. After that, you can run `make` or run any of the tests individually with `./test-name.sh -v`.

**Comment by RichardLitt** [comment]
Updated. Only failures were in the docker tests, which were unrelated - not sure why those aren't working.

**Pull request merged**